impl/src/main/resources/com/okta/authn/sdk/version.properties

```diff
@@ -14,4 +14,4 @@
 # limitations under the License.
 #
 
-${project.version}
\ No newline at end of file
+okta-auth-java=${project.version}
```

Comment:
Pull this one out into a different PR and make sure we have a test for it.
IIRC this file isn't used anymore, as it _should_ be in META-INF

Reply:
sure, will do

Reply:
dropped this change from this PR.